res/css/views/dialogs/_DevtoolsDialog.scss

```diff
@@ -135,6 +135,7 @@ limitations under the License.
     }
 }
 
+/* stylelint-disable no-descending-specificity */
```

Comment:
A comment to explain why this is here would be nice

Reply:
Agreed, added comments.